.github/workflows/format-lint-test.yaml

```diff
@@ -0,0 +1,31 @@
+name: Format, Lint, Test
+
+on:
+  push:
+    branches: [ "dev" ]
+  pull_request:
+    branches: [ "dev" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Use Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '20.x'
+    - name: Install dependencies
+      working-directory: ./unime
+      run: npm install    
+    - name: Format
+      working-directory: ./unime
+      run: npm run format:check
+    # - name: Lint
+    #   working-directory: ./unime
+    #   run: npm run lint
+    - name: Test (frontend)
+      working-directory: ./unime
+      run: npx vitest run
```

Comment:
`npm run test` runs all tests in watch mode by default (does not terminate after successful test run). I can also add a `test:run` npm script, but I chose to execute vitest directly. It will later be replaced by `npm run coverage` which terminates _and_ adds a test coverage report.